homeassistant/components/toon/binary_sensor.py

```diff
@@ -104,3 +81,12 @@ class ToonDisplayBinarySensor(ToonBinarySensor, ToonDisplayDeviceEntity):
 
 class ToonBoilerModuleBinarySensor(ToonBinarySensor, ToonBoilerModuleDeviceEntity):
     """Defines a Boiler module binary sensor."""
+
+
+# pylint: disable=wrong-import-position
+# Necessary to prevent circular import
+from .entity_descriptions import (  # noqa: E402
```

Comment:
I will fix this later... (if that is needed, as it seems like the developer program is being shut down).